README.md

```diff
@@ -256,4 +256,6 @@ Each time you call `get()` on the injected `Provider`, the configuration propert
 
 # License
 
-[Apache License 2.0](LICENSE)
+This code pattern is licensed under the Apache Software License, Version 2.  Separate third party code objects invoked within this code pattern are licensed by their respective providers pursuant to their own separate licenses. Contributions are subject to the [Developer Certificate of Origin, Version 1.1 (DCO)](https://developercertificate.org/) and the [Apache Software License, Version 2](http://www.apache.org/licenses/LICENSE-2.0.txt).
```

Comment:
I am not sure why people insist on calling it "Apache Software License" when it is clearly called "Apache License".  Please use the correct license name.